main.go

```diff
@@ -185,6 +257,21 @@ func main() {
 			if installPage != "" {
 				publicInstallPages[filepath.Base(pth)] = installPage
 			}
+		case ".zip":
+			if xcarchive.CheckForXcarchive(pth) {
+				log.Donef("Uploading xcarchive file: %s", pth)
+
+				installPage, err := uploaders.DeployXcarchive(pth, config.BuildURL, config.APIToken, config.NotifyUserGroups, config.NotifyEmailList, config.IsPublicPageEnabled)
+				if err != nil {
+					fail("Deploy failed, error: %s", err)
+				}
+
+				if installPage != "" {
+					publicInstallPages[filepath.Base(pth)] = installPage
+				}
+				break
+			}
+			fallthrough
```

Comment:
fallthrough is for running all the switch cases below the current case, please do not use it if it is not required

Reply:
unfortunately fallthrough doesn't mean to "jump to the next case", it means "execute the rest of the case code blocks you can see under me"

Reply:
Removed.